components/style/properties/gecko.mako.rs

```diff
@@ -1727,15 +1727,13 @@ fn static_assert() {
 
     pub fn set_animation_name(&mut self, v: longhands::animation_name::computed_value::T) {
         use nsstring::nsCString;
+
+        assert!(v.0.len() > 0);
```

Comment:
I follow lots of `assert!` around this file in the same format... probably we can convert all of them into `debug_assert!(!v.0.is_empty())` in a separate commit.